libs/design/select/examples/src/default-select/default-select.component.html

```diff
@@ -0,0 +1,26 @@
+<daff-select [options]="options" [formControl]="control">
+  <ng-template daffSelectSelectedOption let-selected="selected">
+    <div *ngIf="!selected">
+      Select an address
```

Comment:
Can we use enforce the usage of  <label daffFormLabel>  in the component for the placeholder?